### PR TITLE
Switched ladspaeffect/Cmakelists.txt to use `LMMS_HAVE_*` instead of `WANT_*`

### PR DESCRIPTION
**Pull request by Rossmaxx**
per https://github.com/LMMS/lmms/pull/6758#discussion_r1336190915

I decided to open a seperate pr instead of adding it to #6758 for easier merge.

### CODE REVIEW

File: plugins/LadspaEffect/CMakeLists.txt [hunk 1]
```diff
@@ -4,22 +4,22 @@ BUILD_PLUGIN(ladspaeffect LadspaEffect.cpp LadspaControls.cpp LadspaControlDialo
...
 SET(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/ladspa")
 
-IF(WANT_CAPS)
+IF(LMMS_HAVE_CAPS)
 ADD_SUBDIRECTORY(caps)
-ENDIF(WANT_CAPS)
+ENDIF()
 
-IF(WANT_TAP)
+IF(LMMS_HAVE_CAPS)
```
Comment (DomClark) [inline plugins/LadspaEffect/CMakeLists.txt:11]:
```suggestion
IF(LMMS_HAVE_TAP)
```

File: plugins/LadspaEffect/CMakeLists.txt [hunk 2]
```diff
@@ -4,22 +4,22 @@ BUILD_PLUGIN(ladspaeffect LadspaEffect.cpp LadspaControls.cpp LadspaControlDialo
...
+ENDIF()
 
-IF(WANT_SWH)
+IF(LMMS_HAVE_SWH)
 ADD_SUBDIRECTORY(swh)
-ENDIF(WANT_SWH)
+ENDIF()
 
-IF(WANT_CMT)
+IF(LMMS_HAVE_SWH)
```
Comment (DomClark) [inline plugins/LadspaEffect/CMakeLists.txt:19]:
```suggestion
IF(LMMS_HAVE_CMT)
```

File: plugins/LadspaEffect/CMakeLists.txt [hunk 3]
```diff
@@ -4,22 +4,22 @@ BUILD_PLUGIN(ladspaeffect LadspaEffect.cpp LadspaControls.cpp LadspaControlDialo
...
 ADD_SUBDIRECTORY(cmt)
-ENDIF(WANT_CMT)
+ENDIF()
 
-IF(WANT_CALF)
+IF(LMMS_HAVE_SWH)
```
Comment (DomClark) [inline plugins/LadspaEffect/CMakeLists.txt:23]:
```suggestion
IF(LMMS_HAVE_CALF)
```

File: plugins/LadspaEffect/CMakeLists.txt (hunk 2 above)
Comment (Rossmaxx) [reply plugins/LadspaEffect/CMakeLists.txt:19]: Whoops, messed up the order. Thanks for catching.

**Review by DomClark — APPROVED** [review]

**Review by sakertooth — APPROVED** [review]

**Pull request merged**